data/moves.ts

```diff
@@ -13194,8 +13194,11 @@ export const Moves: {[moveid: string]: MoveData} = {
 		pp: 10,
 		priority: 0,
 		flags: {protect: 1, pulse: 1, mirror: 1},
+		onUseMoveMessage(source, target, move) {
```

Comment:
I'll guess this meant to be in a different PR, but the current behavior already does announce Armor Tail so I'm confused what this is doing. https://replay.pokemonshowdown.com/gen9doublescustomgame-1743581589-gb8d45dyyiz3xc85csvvfp5eax1v6v7pw